discord-canary.sh

```diff
@@ -5,5 +5,14 @@ socat $SOCAT_ARGS \
     UNIX-CONNECT:$XDG_RUNTIME_DIR/discord-ipc-0 \
     &
 socat_pid=$!
-env TMPDIR=$XDG_CACHE_HOME CHROME_WRAPPER=$(readlink -f "$0") zypak-wrapper.sh /app/discord-canary/DiscordCanary "$@"
+
+FLAGS='--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --enable-features=UseSkiaRenderer,CanvasOopRasterization'
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -c /dev/nvidia0 ]
+then
+    FLAGS="$FLAGS --disable-gpu-sandbox"
```

Comment:
Do you have the source for where this was needed? Sounds like something that should be filed upstream if it isn't already...